### ISSUE TITLE
md-tab loaded from ng-include getting chopped off horizontally

### ISSUE THREAD

**Issue by opadillaintegrify** [angular/material#2469]
```
<md-tabs class="md-accent" md-align-tabs="top" md-dynamic-height>
    <md-tab id="tab1" label="Request Detail">

        <md-tab-body>
            <div ng-include src="requestSummary.Process.StyleTemplateUrl"></div>
        </md-tab-body>
    </md-tab>
   ...
</md-tabs>
```
It fit using md-tabs in version 0.8.3.

**Comment by opadillaintegrify** [comment]
I just noticed that this problem only happens in Chrome Canary (ver. 44.0.2378.0), which is the browser I use for development, and Firefox.  

Works fine in regular Chrome and Safari.  No idea what this does in IE.

**Comment by patriknyd** [comment]
+1
See the same issue in IE 11.9600.17728 on Win 8.1 x64. 
Works in FF 37.0.2 and Chrome 42.0.2311.90.

**Comment by robertmesserle** [comment]
@patriknyd @opadillaintegrify Is this still an issue with the `0.9.8-rc1`?

**Comment by robertmesserle** [comment]
Closing for now, please respond if this is still an issue and I will reopen.  If possibly, include a CodePen or Plunker in your comment.

**Comment by patriknyd** [comment]
No, this is not an issue for me any more. Thanks.

On Mon, Jun 8, 2015 at 8:39 PM, Robert Messerle <notifications@github.com>
wrote:

> Closed #2469 <https://github.com/angular/material/issues/2469>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/angular/material/issues/2469#event-325576778>.
>



-- 

/ *Patrik Nydensten*
